frontend/public/src/lib/util.js

```diff
@@ -262,12 +262,18 @@ export const getFlexiblePriceForProduct = (product: Product) => {
 
 export const intCheckFeatureFlag = (
   flag: string,
+  uniqueID: any,
```

Comment:
Nit: could this be `string|number` rather than `any`? (Also on line 285.)

Reply:
fair point - I think it was nullable in my mind, and that was my way of allowing all 3 rather than string|number. Bad plan. I tweaked it to specify both and also to not have the extraneous if statement - we are declaring there is a uniqueID so it will blow up before that point if there is not... and in some cases could evaluate to false there when it should not (it's fine to send an empty user_id... not ideal, but won't break posthog).